tests/sentry/monitors/endpoints/test_monitor_ingest_checkin_index.py

```diff
@@ -376,3 +380,24 @@ def test_rate_limit(self):
                     path, {"status": "ok", "environment": "dev"}, **self.token_auth_headers
                 )
                 assert resp.status_code == 429, resp.content
+
+    def test_bad_config(self):
+        for path_func in self._get_path_functions():
+            monitor = self._create_monitor()
+            monitor.config = {
+                "schedule": "* * * * *",
+                "schedule_type": ScheduleType.CRONTAB,
+            }
```

Comment:
What makes this a bad config?

Reply:
its missing checkin_margin, and max_runtime which should always be set

Reply:
Do we actually always set them?

Reply:
I guess they are set, but just `None`, and from what I saw I couldn't find a scenario where they weren't there in the config object. I can remove another field in this config tho to make it more obviously bad.